docs/it-manual/sre-playbook/server-environment-variables/README.md

```diff
@@ -10,3 +10,11 @@ corresponding to the release is used to generate markdown documentation of the
 variables. The generated markdown files are added to this directoy. Use the
 left sidebar to see the available configuration variables for the CiviForm
 version you are deploying.
+
+Variables can have four different modes which determine where they are set and displayed:
```

Comment:
This is external facing documentation, so I want to use the phrasing we use in our external var documentation eg: https://docs.civiform.us/it-manual/sre-playbook/upgrading-to-a-new-release/server-environment-variables/v1.51.0

It took me a minute to translate them too, but I think we should avoid using the modes we reference in `env-var-docs.json` because that could be confusing. Here's where they're translated for the documentation: https://github.com/civiform/civiform/blob/ae755a1c3f6c2f5f94ee2584965d7a12b3645dc8/env-var-docs/generate_markdown.py#L167.

That said, I think adding quotes around the modes would help clear up what I'm actually referencing.